src/main/java/net/sacredlabyrinth/phaed/simpleclans/commands/clan/ClanCommands.java

```diff
@@ -44,21 +44,21 @@
     @Conditions("verified|rank:name=WAR_START")
     @Description("{@@command.description.war.start}")
     @CommandCompletion("@rivals")
-    public void startWar(Player player, ClanPlayer cp, Clan requesterClan, @Conditions("can_war_target") @Name("clan") ClanInput targetClanInput) {
+    public void startWar(Player player, ClanPlayer requester, Clan requestClan, @Conditions("can_war_target") @Name("clan") ClanInput targetClanInput) {
```

Comment:
Do we need to have `Player player` here?
Why just don't requester.toPlayer()?

Reply:
requester.toPlayer() = more method calls.
Player as a parameter looks cleaner.

Reply:
👌🏻